postgraphile/README.md

```diff
@@ -23,8 +23,7 @@ Build the docker image in this directory. Start the `GraphiQL` frontend by:
 * GraphiQL frontend is available at `:3000/graphiql`
   GraphQL endpoint is available at `:3000/graphql`
 
-By default, this build will expose only the "public" schema - to add other schemas, use either the env variables,
-or a config file `config.toml` and set the env var `CONFIG_PATH` to point to its location. Example `toml`:
+By default, this build will expose only the "public" schema and will disable mutations - to change mutation behaviour, you can use an optional config file `config.toml` and set the env var `CONFIG_PATH` to point to its location. Example `toml`:
```

Comment:
This isn't specific to this PR since `CONFIG_PATH` was introduced earlier, but I wonder if we should use a more specific env variable name for this? How "hard-coded" is this in Postgraphile? Seems unlikely but `CONFIG_PATH` is pretty generic and could potentially run into a conflict.